src/share-buttons.js

```diff
@@ -26,7 +26,7 @@
             IN_LINK_FORMAT = 'https://www.linkedin.com/shareArticle?mini=true&url={0}&title={1}&summary={2}&source={0}',
             PI_LINK_FORMAT = 'https://pinterest.com/pin/create/button/?url={0}&media={0}&description={1}',
             SK_LINK_FORMAT = 'https://web.skype.com/share?url={0}&source=button&text={1}',
-            WA_LINK_FORMAT = 'whatsapp://send?text={0}%20{1}',
+            WA_LINK_FORMAT = 'https://wa.me/send?text={0}%20{1}',
```

Comment:
Based on WhatsApp doc:
https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/?lang=en

```suggestion
            WA_LINK_FORMAT = 'https://wa.me/?text={0}%20{1}',
```